NEWS.md

```diff
@@ -1,6 +1,31 @@
 erGW - 3GPP GGSN and PDN-GW in Erlang
 =====================================
 
+Version 2.6.0 - 01 October 2020
+-------------------------------
+
+* feature: provide implementation for `DELETE /api/v1/contexts/:count` for
+  GTP-Proxy setups ("drain")
+* feature: configurable TEID prefix to use
+* feature: async IP assignment for IPv4 and IPv6
+* feature: simple cache for Node Selection results based upon DNS queries
+
+* fix handling DNS responses without IP in Sx node selection
+  (would shutdown of *ergw*)
+* fix Change Reporting Indication/Action (Offline Charging)
+* fix encoding of IE 'Indication'
+* improve GTP Path Maintenance procedures.
+  NOTE: DESCRIBE KNOWN ISSUE
```

Comment:
@fholzhauser please describe the known issue and how to mitigate via path-maintenance-config values